subworkflows/local/parse_input.nf

```diff
@@ -4,23 +4,23 @@
 
 params.options = [:]
 
-include { parse_samplesheet } from '../../modules/local/parse_samplesheet' addParams( options: params.options )
+include { parse_samplesheet } from '../../bin/parse_samplesheet' addParams( options: params.options )
```

Comment:
Again I'm not sure if bin is the best location. Why not adding the nf code directly as part of the `PARSE_INPUT` subworkflow file?

Reply:
Done, right, thats better!